### ISSUE TITLE
Postinstall issues

### ISSUE THREAD

**Issue by OldShaterhan** [simple-icons/simple-icons-font#93]
Hi,

I'm using `simple-icons-font` in one of projects.

Unfortunately, after update to 4.6.0 and npm install, each time I'm getting:
```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! simple-icons-font@4.6.0 postinstall: `is-ci || husky install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the simple-icons-font@4.6.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\tomek\AppData\Roaming\npm-cache\_logs\2021-01-16T02_55_59_755Z-debug.log
```

This wasn't issue for 4.5.0, but after upgrade it started. I'm using postinstall in my project.

**Comment by mondeja** [comment]
I can replicate the bug, hopefully #94 should fix it.

**Comment by ericcornelissen** [comment]
Oops, can't believe we forgot about this, [the husky installation guide even talks about this](https://typicode.github.io/husky/#/?id=install)...

**Comment by ericcornelissen** [comment]
Just [released v4.6.1](https://github.com/simple-icons/simple-icons-font/pull/95) and everything seems fine on my end when I install that version as a dependency, let us know if you're still having trouble @OldShaterhan.

**Comment by OldShaterhan** [comment]
Thanks, I will test it in the following days.
